lib/validation.js

```diff
@@ -109,7 +109,9 @@ function validateOpts (app, opts = {}) {
         if (policyField.references && typeof policyField.references !== 'function') {
           throw new Error(`policy '${type}.${name}' references must be a function`)
         }
-        maxTTL = Math.max(maxTTL, policyField.ttl)
+        if (!isNaN(policyField.ttl)) {
```

Comment:
this check should be added at https://github.com/mercurius-js/cache/blob/1e905afe6a08d35f585ec414fb22bb70db9d8824/lib/validation.js#L84

Reply:
thanks @simone-sanfratello I hope this is ok?

Reply:
see the last commit, please write test accordingly